packages/compass-web/src/logger-and-telemetry.tsx

```diff
@@ -11,83 +9,18 @@ export type TrackFunction = (
   properties: Record<string, any>
 ) => void;
 
-export type LogFunction = (
-  type: 'debug' | 'info' | 'warn' | 'error' | 'fatal',
-  ...args: any[]
-) => void;
+export type LogFunction = (message: {
+  id: number;
```

Comment:
So after this I can use ids to match specific events? Just making sure that's the expectation

Reply:
Good call, ty!